.gitignore

```diff
@@ -5,3 +5,4 @@ libcgotorch.so
 *~
 .vscode
 data
+*.test
```

Comment:
Need this line?

Reply:
Because in the root directory, all *_test.go files define a new package gotorch_test.  This new package would cause go test to build a new binary ./gotorch.test. We want to ignore this binary file.